### ISSUE TITLE
First few letters of stat select when searching for items/stats are cut off

### ISSUE THREAD

**Issue by lightyearsaway** [dofuslab/dofuslab#100]
![](https://media.discordapp.net/attachments/694305331232112791/705213807017852978/image0.jpg?width=511&height=910)

**Comment by paddykontschak** [comment]
I'm still investigating this, but as of right now this seems like an iOS Safari-only issue.

I noticed this in `antd`'s styles:
```css
.ant-select-multiple .ant-select-selection-search-input {
  width: 100%;
  min-width: 3px;
}
```
When I set `min-width` to `max-content` it fixes this issue until it reaches around 2/3 of the width of the input and then it starts hiding the first couple letters - too early for normal input behavior.

**Comment by lightyearsaway** [comment]
Fixed in #184 by upgrading antd
